### PR TITLE
Push module now creates maven artifacts;

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

File: push/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,3 @@
+POM_NAME=AeroGear Android SDK
+POM_ARTIFACT_ID=push
```
Comment (danielpassos) [inline push/gradle.properties]: Let's use aerogear-push like in Matthias PR https://github.com/aerogear/aerogear-android-sdk/pull/144

**Comment by secondsun** [comment]
@danielpassos done

**Comment by danielpassos** [comment]
@secondsun rebase, merge and call it a day!

**Comment by secondsun** [comment]
@danielpassos rebased can you merge?

**Pull request merged**